bql/planner/planner_test.go

```diff
@@ -776,18 +691,10 @@ func TestPlannerQuery(t *testing.T) {
 
 		// Actual test:
 		tbl, err := plnr.Execute(ctx)
-		if !entry.wantErr && err != nil {
+		if err != nil {
 			t.Errorf("planner.Execute(%s)\n= _, %v; want nil error", entry.q, err)
 			continue
 		}
-		if entry.wantErr && err == nil {
-			t.Errorf("planner.Execute(%s)\n= _, nil; want non-nil error", entry.q)
-			continue
-		}
-		if entry.wantErr {
-			// an error was expected and it was indeed caught.
-			continue
```

Comment:
So good to see this gone! Thanks for the refactor! :D

Reply:
hahaha :D